### DISCUSSION TITLE
Screen shaders / Postprocessing

### DISCUSSION THREAD

**Discussion by voxelias** [gfx-rs/wgpu#2114] [Q&A]
Hi!

Is it possible to implement screen shaders and post processing effects using WGPU?

I mean when you apply some effect to everything that us rendered, like this for example:
![CRT](https://www.alanzucconi.com/wp-content/uploads/2015/06/crt.gif)

Thanks in advance

**Comment by cwfitzgerald** [comment]
Absolutely! The way you'd do it depends a lot on the effect you need, but whatever you'd want to do, it's possible.

**Reply by voxelias** [reply]
Could you please recommend me some entry point? Some example? Or specific library components I should investigate?

**Reply by CatCode79** [reply]
Check out these two libraries:
https://github.com/fintelia/smaa-rs
https://github.com/kocsis1david/wgrepp

(sadly they use shaderc instead of naga, but it's still a start)

**Reply by voxelias** [reply]
Thank you! Exactly what I need!
